holoviews/plotting/bokeh/chart.py

```diff
@@ -45,7 +46,7 @@ class PointPlot(LegendPlot, ColorbarPlot):
       Function applied to size values before applying scaling,
       to remove values lower than zero.""")
 
-    style_opts = (['cmap', 'palette', 'marker', 'size', 's', 'alpha', 'color',
+    style_opts = (['cmap', 'palette', 'marker', 'size', 's', 'color',
```

Comment:
I feel ``style_opts`` should be a set - something for 2.0 perhaps? I am assuming 'alpha' was being duplicated...